src/test/java/org/springframework/data/couchbase/repository/ReactiveCouchbaseRepositoryQueryIntegrationTests.java

```diff
@@ -128,8 +130,11 @@ void count() {
 				Airport airport = new Airport("airports::" + iatas[i], iatas[i] /*iata*/, iatas[i].toLowerCase() /* lcao */);
 				airportRepository.save(airport).block();
 			}
-
-			Long airportCount = airportCount = airportRepository.count().block();
+			List<Airport> airports = airportRepository.findAll().collectList().block();
+			for (Airport ap : airports) {
+				System.out.println(ap);
```

Comment:
is the println needed?

Reply:
Not anymore.  There were 8 airports after inserting 7.  The println was to find the extra one - it was was because the deleteAll() in an earlier test did not block.  I'll remove the whole findAll()/for